community/asking-questions.md

```diff
@@ -0,0 +1,25 @@
+# Asking Questions on the Forum
+
+Author(s): code3z (@code3 on the forum)
+
+The [modding forum](https://forum.vivaldi.net/category/52/modifications)
+is also a great place to ask questions about modding. Before asking, search the
+forum and the internet to check if there is not already a good answer.
+
+When asking a question, keep a few simple things in mind:
+ - Use a topic title that is descriptive
+ - Be polite and don't use ALL CAPS
+ - If you are asking questions about a certain piece of code, share that code
+ - Share any ways you thought you might fix the problem or things that didn't work
+ - Don't add the "mod" tag to your post - save that for when you actually publish a mod
+
+## Asking For mods
```

Comment:
```suggestion
## Asking for Mods
```
(yes, I’ll go with the standard rules – but this didn’t follow even them)

Please check the other headings as well.